.github/publish-release-draft/README.md

```diff
@@ -0,0 +1,3 @@
+# Release Draft
+
+Creates pre-releases with release notes draft and publishes it for viewing
```

Comment:
Please add more details
- Where do the notes come from? and how?
- Where is the draft published?